lib/originals-helper.js

```diff
@@ -22,8 +22,12 @@ function nodeInTSDefs(node) {
  * @return {boolean} whether the type of the node matches IDL in a web spec
  */
 function nodeInIDLSpecs(node) {
-  return globalsSet.has(
-      node.getType().getSymbol().getName());
+  const symbol = node.getType().getSymbol();
```

Comment:
Why was this change introduced in this PR?

Reply:
Will move to refactor PR